src/store/lokapi.ts

```diff
@@ -63,20 +63,28 @@ export function lokapiStoreFactory(lokApiService: any) {
       },
       async fetchAccounts({commit}:any) {
         commit("setAccountsLoading", true)
-        const { virtualAccountTree, allMoneyAccounts } = await lokApiService.buildVirtualAccountTree()
-        commit("setAccounts", { virtualAccountTree, allMoneyAccounts })
+        try {
+          const { virtualAccountTree, allMoneyAccounts } = await lokApiService.buildVirtualAccountTree()
+          commit("setAccounts", { virtualAccountTree, allMoneyAccounts })
+        } catch (e:any) {
+          console.error('Error fetching wallets', e.message, "\nDebug: " + JSON.stringify(e))
+        }
         commit("setAccountsLoading", false)
       },
       async resetTransactions({commit, dispatch, state}:any) {
         commit("setTransactionsLoading", true)
-        transactionsGen = lokApiService.getTransactions()
         let transactions = [],
             transactionsIndex = 0
-        let next = await transactionsGen.next()
-        while (!next.done && transactionsIndex < transactionsBatchLength) {
-          transactionsIndex++
-          transactions.push(<any>next.value)
-          next = await transactionsGen.next()
+        try {
+          transactionsGen = lokApiService.getTransactions()
+          let next = await transactionsGen.next()
+          while (!next.done && transactionsIndex < transactionsBatchLength) {
+            transactionsIndex++
+            transactions.push(<any>next.value)
+            next = await transactionsGen.next()
+          }
+        } catch (e:any) {
+          console.error('Error fetching transactions', e.message, "\nDebug: " + JSON.stringify(e))
```

Comment:
Same remark here than previously.
Note that the console.error message isn't adding much as the exception will write its own message if not caught. We can still give the caller the choice of displaying in the console this message or not.

Reply:
Hmmm. `resetTransactions` action is also called from the store. So the caller is the store. The store decided to write the message to the js console. As stated below, it's a tradeof, not ideal I agree, but unless we tie all the action calls in components I think that's the way to do it.